MapboxNavigationUI/RouteVoiceController.swift

```diff
@@ -242,16 +242,23 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
         return text
     }
     
-    func localizeRoadDescription(_ step: RouteStep) -> String {
+    func localizeRoadDescription(_ step: RouteStep, markUpWithSSML: Bool) -> String {
         var road = ""
+        let escapeIfNecessary = {(distance: String) -> String in
+            return markUpWithSSML ? distance.addingXMLEscapes : distance
+        }
         if let name = step.names?.first {
             if let code = step.codes?.first {
-                road = String.localizedStringWithFormat(NSLocalizedString("NAME_AND_REF", value: "%@ (%@)", comment: "Format for speech string; 1 = way name; 2 = way route number"), name, code)
+                if markUpWithSSML {
+                    road = String.localizedStringWithFormat(NSLocalizedString("NAME_AND_REF", value: "<say-as interpret-as=\"address\">%@</say-as> (<say-as interpret-as=\"address\">%@</say-as>)", comment: "Format for speech string; 1 = way name; 2 = way route number"), name.addingXMLEscapes, code.addingXMLEscapes)
```

Comment:
This line cannot have the parens inside the markup https://github.com/mapbox/MapboxNavigation.swift/blob/7aeaff76c175183e3b5b6b75a22d59a811c7b31e/MapboxNavigationUI/RouteVoiceController.swift#L253